tests/content/src/main/content/jcr_root/content/howlite-test/pages/Accordion/.content.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:ws="http://ds.pl/websight" xmlns:sling="http://sling.apache.org/jcr/sling/1.0"
+    jcr:primaryType="ws:Page">
+    <jcr:content
+        jcr:primaryType="ws:PageContent"
+        jcr:title="Accordion"
+        sling:resourceType="howlite/components/page"
+        ws:template="/libs/howlite/templates/contentpage">
+        <rootcontainer
+            jcr:primaryType="nt:unstructured"
+            sling:resourceType="howlite/components/rootcontainer/fixedrootcontainer">
+            <header
+                jcr:primaryType="nt:unstructured"
+                sling:resourceType="howlite/components/headerreference"/>
+            <maincontainer
+                jcr:primaryType="nt:unstructured"
+                sling:resourceType="howlite/components/maincontainer">
+                <pagesection
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="howlite/components/pagesection">
+                    <accordion
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="howlite/components/accordion">
+                        <accordionitem1
+                            jcr:primaryType="nt:unstructured"
+                            sling:resourceType="howlite/components/accordion/accordionitem"/>
+                        <accordionitem2
+                            jcr:primaryType="nt:unstructured"
+                            sling:resourceType="howlite/components/accordion/accordionitem"/>
+                        <accordionitem3
+                            jcr:primaryType="nt:unstructured"
```

Comment:
let's fill properties in one of the accordion items. 
Also - please add one empty Accordion below (with all accordion items removed) - we need to make sure that the accordion placeholder is also rendered correctly